src/views/EditorPage.vue

```diff
@@ -119,6 +119,8 @@ export default {
     }
   },
   mounted() {
+    this.$store.dispatch('setUser')
```

Comment:
maybe we don't need to call setUser when app is mounted, just save the user meta to localStorage after login?